language-support/ts/codegen/tests/ts/daml-json-types/src/index.ts

```diff
@@ -112,6 +112,26 @@ export const Decimal: Serializable<Decimal> = {
   decoder: jtv.string,
 }
 
+/**
+ * The counterpart of DAML's `Numeric` type. We represent `Numeric`s as string
+ * in order to avoid a loss of precision. The string must match the regular
+ * expression `-?(?:0|[1-9]\d*)(?:\.\d+)?(?:[eE][+-]?\d+)?`.
+ */
+export type Numeric = string;
+
+/**
+ * Companion function of the `Numeric` type.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const Numeric = function (_: number): Serializable<Numeric> {
```

Comment:
That is a quite peculiar style for defining a function. We should either go for `export function Numeric(_: number): Serializable<Numeric> { ... }` or for `export const Numeric = (_: number): Serializable<Numeric> => ...`. To fit the style of the surrounding code, I suggest
```suggestion
export const Numeric = (_: number): Serializable<Numeric> => ({
  decoder: jtv.string,
});
```

Reply:
Will fat arrow this.

Reply:
✔️